packages/vuexfire/examples/TodoApp/modules.html

```diff
@@ -60,12 +60,10 @@
             itemsRef.doc(id).delete()
           },
           addTodo: function() {
-            if (this.newTodo.trim()) {
-              itemsRef.add({
-                text: this.newTodo,
-              })
-              this.newTodo = ''
-            }
+            itemsRef.add({
```

Comment:
We still need to keep the `if (this.newTodo)`

Reply:
fixed on latest commit (apologies!)